HW6.py

```diff
@@ -0,0 +1,56 @@
+import xml.etree.ElementTree as ET
+import re
+
+
+def parse_and_remove(filename, path):
+    path_parts = path.split('/')
+    doc = ET.iterparse(filename, ('start', 'end'))
+    # Skip root element
+    next(doc)
+    tag_stack = []
+    elem_stack = []
+    for event, elem in doc:
+        if event == 'start':
+            tag_stack.append(elem.tag)
+            elem_stack.append(elem)
+        elif event == 'end':
+            if tag_stack == path_parts:
+                yield elem
+            try:
+                tag_stack.pop()
+                elem_stack.pop()
+            except IndexError:
+                pass
+#code for task 1 Country name and goverment with out duplicates
+
+lines_seen = set()
+country_government = []
+countries = parse_and_remove('mondial-3.0.xml', 'country')
+for country in countries:
+    government = country.attrib['government'].split(' ')[0]
```

Comment:
A little bit more appropriate way to remove spaces from the left side:

`government = country.attrib['government'].lstrip()`